src/plugins/Dashboard/index.js

```diff
@@ -13,19 +13,22 @@ const { defaultTabIcon } = require('./icons')
 // MIT licence, https://github.com/ghosh/micromodal/blob/master/LICENSE.md
 // Copyright (c) 2017 Indrashish Ghosh
 const FOCUSABLE_ELEMENTS = [
-  'a[href]',
-  'area[href]',
-  'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
-  'select:not([disabled]):not([aria-hidden])',
-  'textarea:not([disabled]):not([aria-hidden])',
-  'button:not([disabled]):not([aria-hidden])',
-  'iframe',
-  'object',
-  'embed',
-  '[contenteditable]',
-  '[tabindex]:not([tabindex^="-"])'
+  'a[href]:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  'area[href]:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  'input:not([disabled]):not([inert]):not([aria-hidden])',
+  'select:not([disabled]):not([inert]):not([aria-hidden])',
+  'textarea:not([disabled]):not([inert]):not([aria-hidden])',
+  'button:not([disabled]):not([inert]):not([aria-hidden])',
+  'iframe:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  'object:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  'embed:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  '[contenteditable]:not([tabindex^="-"]):not([inert]):not([aria-hidden])',
+  '[tabindex]:not([tabindex^="-"]):not([inert]):not([aria-hidden])'
```

Comment:
good improvements :+1: Does the `:not([inert])` thing actually do anything? I thought any children of `[inert]` elements should also not be interactive, that's not possible to do with CSS selectors though.

Reply:
It’s just here in case we do have `inert` element, and then it will not be selectable via TAB? Inert idea came from https://github.com/edenspiekermann/a11y-dialog/blob/c1cb2dbf2aa866a50e34f5ea082e5e9f77ded2f7/a11y-dialog.js

Reply:
I guess, my impression is that `inert` is most useful if eg. you have an overlay panel that is always mounted but not always visible, you can set `inert` on it to disable every focusable element inside it. It doesn't hurt for it to be there like this either tho.